### PR TITLE
Update sftp.md to remove filezilla

### PR DESCRIPTION
**Pull request by JamesH65**
Filezilla for windows appears to install malware, remove the recomendation and substitute with winscp.

### CODE REVIEW

**Comment by lurch** [comment]
Should this page also recommend a Mac SFTP client?

**Comment by JamesH65** [comment]
Probably, but I know nothing about Macs.....

**Comment by lurch** [comment]
Me neither :wink: 
Ping @spl237

**Comment by spl237** [comment]
Err - pass. Never used sftp on the Mac - but both scp and sftp are already installed as command-line apps on the Mac and can be used from the Terminal.

**Comment by andrum99** [comment]
I would personally question including the Linux version of Filezilla in the docs, given they are happy to include dubious binaries in their Windows installer. It's also not really necessary to recommend a GUI sftp client for Linux, IMHO. (Also, sftp URL's Just Work (TM) in the Pi desktop's file manager 😀).

File: remote-access/ssh/sftp.md [hunk 1]
```diff
@@ -4,9 +4,13 @@ The SSH File Transfer Protocol is a network protocol that provides file access,
 
 By using SFTP, you can easily change, browse, and edit files on your Raspberry Pi. SFTP is easier to set up than [FTP](../ftp.md) once Raspbian has SSH enabled. For security reasons, since the November 2016 release of Raspbian, the SSH server has been disabled by default. To enable it, please follow [these instructions](https://www.raspberrypi.org/documentation/remote-access/ssh/README.md).)
 
-## FileZilla
+## WinSCP on Windows
 
-Download the latest FileZilla Client version for your operating system from [filezilla-project.org](https://filezilla-project.org/).
+We recommend using the [WinSCP FTP client](https://winscp.net/eng/index.php). Follow the instructions on the WinSCP website to install the client, then follow the [WinSCP Quick Start instructions](https://winscp.net/eng/docs/getting_started).
```
Comment (lurch) [inline remote-access/ssh/sftp.md]: This should probably say "WinSCP SFTP client" ?

Comment (JamesH65) [reply remote-access/ssh/sftp.md]: Fixed

**Pull request merged**